NDB.Covid19/NDB.Covid19.iOS/Views/Welcome/ChildViews/WelcomePageTwoViewController.cs

```diff
@@ -17,7 +17,9 @@ public override void ViewDidLoad()
             base.ViewDidLoad();
             SetTexts();
 
-            PageTitle.AccessibilityLabel = WelcomeViewModel.WELCOME_PAGE_TWO_ACCESSIBILITY_TITLE;
+            PageTitle.AccessibilityLabel = WelcomeViewModel.ANNOUNCEMENT_PAGE_CHANGED_TO_TWO;
+            PageTitle.AccessibilityValue = WelcomeViewModel.WELCOME_PAGE_TWO_TITLE;
+            PageTitle.AccessibilityTraits = UIAccessibilityTrait.Header;
```

Comment:
I would prefer to put these in the deleted SetAccessibility() method :) Why don't you wanna do it this way?

Reply:
Changed